utils/ebpf/test.cpp

```diff
@@ -0,0 +1,95 @@
+#include <cstdint>
+#include <iostream>
+#include <cstdlib>
+#include <cstring>
+#include <cassert>
+
+extern "C" {
+    long bpf_get_current_comm(void *buf, uint32_t size_of_buf);
+    int64_t nd_int() {
+        return 0; 
+    }
+
+    long nd_long() {
+        return 0; 
+    }
+
+    uint64_t nd_u64() {
+        return 0; 
+    }
+
+    uint32_t nd_u32() {
+        return 0; 
+    }
+
+    int64_t memhavoc(void *ptr, int size) {
```

Comment:
better implementation is to `memset` the memory with some random or even just 0 value